GVRf/Sample/scene-objects/src/org/gearvrf/sample/sceneobjects/SampleViewManager.java

```diff
@@ -76,12 +70,17 @@ public void onInit(GVRContext gvrContext) {
         GVRCylinderSceneObject cylinderObject = new GVRCylinderSceneObject(
                 gvrContext);
         GVRConeSceneObject coneObject = new GVRConeSceneObject(gvrContext);
-        GVRTextSceneObject textObject = new GVRTextSceneObject(gvrContext, "Hello World!");
+        GVRTextSceneObject textObject = new GVRTextSceneObject(gvrContext,
```

Comment:
I'll test tomorrow morning, but assuming TextViewSceneObject works as advertised, i'd be totally fine with removing TextSceneObject.